pyccel/decorators.py

```diff
@@ -98,3 +100,7 @@ def allow_negative_index(f,*args):
     def identity(f):
         return f
     return identity
+
+def kernel(f):
+    return numpy.array([[f]])
+
```

Comment:
@EmilyBourne not sure if we can make this available only from the `cuda` module.

Reply:
So instead of doing `from pyccel.decorators import kernel` we would do `from pyccel.cuda.decorators import kernel` or (with a well-defined `__init__.py`) `from pyccel.cuda import kernel`?
This seems ok.

I'm not sure I understand your question "not sure if we can". Do you mean "can" or "should"?

Reply:
when i was writing that code i didn't find a way to do this or maybe i just didn't think enough :sweat_smile: